### ISSUE TITLE
[Feature request] Make clickable area of switches and collapsible sections larger

### ISSUE THREAD

**Issue by MrChocolatine** [mullvad/mullvadvpn-app#6593]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

What I call "switches" is, for example, the setting `Auto-connect`.
What I call "collapsible sections" is, for example, the section `DNS content blockers`.

Considering these 2 examples:
- Currently we can only enable/disable `Auto-connect` by clicking its switch on the right.
- Currently we can only open/close `DNS content blockers` by clicking the arrow on the right.

What I would like is to be able to interact with these settings (and the rest) with a larger area.
To be able to click on the entire entry to interact with it:

Basically, the exact same way the entries from the `Settings` view work => we can click anywhere on them.
Also, it is already working like that in the Android application.

In the following example, red boxes would mean the same area that interact with the setting.
While yellow boxes would still open the information modal.
![mvp](https://github.com/user-attachments/assets/1bdf13b4-848e-4200-a50c-fdea5ef738b6)

### Alternative solutions

None

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [X] Windows
- [X] macOS
- [X] Linux

**Comment by Serock3** [comment]
Thanks for the feedback, I have forwarded it to UI/UX

**Comment by danielbmasud** [comment]
@Serock3 This is something I'm interested in too. I can work on this and have already implemented some of it.

**Comment by Serock3** [comment]
Sure thing, throw us a PR and we'll have a look! 🙂 Ping @raksooo

**Comment by tobias-jarvelov** [comment]
Thanks for the feature request, however we have decided not to implement this feature. We previously rejected an implementation of this feature (#7607) as we were not confident in the user experience of the implementation, but that lack of confidence is also applicable to the feature request as a whole.

If we revisit this feature request we would like to do so with a wider scope, to ensure we cover all our bases with regards to the user experience.
